djangocms_blog/models.py

```diff
@@ -232,7 +232,7 @@ def get_title(self):
         return title.strip()
 
     def get_keywords(self):
-        return self.safe_translation_getter('meta_keywords').strip().split(',')
+        return self.safe_translation_getter('meta_keywords', default='').strip().split(',')
```

Comment:
ps: so, as said above let's make the check on the Post model method get_keywords to have a default '' fallback :+1: .